### PR TITLE
Adding Desktop version

### PR DESCRIPTION
**Pull request by SnakyMz**
Things I have done:

- [ ] Added media query for desktop view
- [ ] Updated Headline section to support responsive desktop viewing
- [ ] Updated Workspace section to support responsive desktop viewing
- [ ] Updated About section to support responsive desktop viewing
- [ ] Updated Contact section to support responsive desktop viewing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,14 +14,20 @@
...
             <a src="" id="logo">My Logo</a>
+            <ul class="menu-items">
+                <li><a href="#workspace">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
             <img src="./images/Enabled.svg" alt="Menu" id="menu">
         </nav>
     </head>
     <main>
```
Comment (stanleySimeon) [inline index.html:25]: Greeting!

Great work so far! But you have a few things to change before moving to the next steps.

- Kindly make sure you stick to desgin as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma Design
>
>Screenshot 2023-06-07 at 5 53 25 PM
>
> ## Your Design
>
>Screenshot 2023-06-07 at 5 44 59 PM
>Screenshot 2023-06-07 at 5 45 16 PM
>Screenshot 2023-06-07 at 5 45 25 PM
>Screenshot 2023-06-07 at 5 45 37 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @SnakyMz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your Readme file
- deploying your project

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (stanleySimeon) [inline index.html:1]: Greeting!

Great work so far, you are doing great work but there are a few things you still need to perform before going to the next steps.

According to the Figma design, I suggest you stick to the design as much as possible `(e.g., font, colors, images, text, margins)` using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).(e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma Design
>
>Screenshot 2023-06-07 at 5 53 25 PM
>
> ## Your Design
>
>Screenshot 2023-06-07 at 5 44 59 PM
>Screenshot 2023-06-07 at 5 45 16 PM
>Screenshot 2023-06-07 at 5 45 25 PM
>Screenshot 2023-06-07 at 5 45 37 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -14,14 +14,20 @@
     <head>
         <nav>
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- [ ] Kindly note that the `head` tag is used before the `body` as you did and it includes hidden information about your website, the `head` tag you used which wraps the `nav` should be named `header` instead, please fix it.

File: index.html [hunk 3]
```diff
@@ -14,14 +14,20 @@
...
             <a src="" id="logo">My Logo</a>
+            <ul class="menu-items">
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- [ ] Kindly note that the link tag `a` does not have a `src` attribute, I believe you meant to use `href` instead.

File: index.html
Comment (Ofarouq310) [inline index.html:1]: [OPTIONAL]
- [ ] Kindly note that use used the same `id` twice, the first on line `34`, and the second on line `152`, which conflicts with the CSS best practices, so please fix it.
- Either change it to a `class`, or use two different `id`s for the two elements.

File: index.html [hunk 4]
```diff
@@ -152,18 +179,21 @@ <h2 class="outro-head">About Myself</h2>
...
+                        project that needs coding. I’d love to help with it!</p>
+                    <form action="https://formspree.io/f/mayzgpbp" method="POST">
+                        <input type="text" name="fullname" maxlength="30" placeholder="Lorem Ipsum" required>
+                        <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                        <textarea name="message" id="message" placeholder="Write your message here..." cols="30"
+                            maxlength="500" rows="6" required></textarea>
+                        <button type="submit" id="submit" class="see-project">Get in touch</button>
+                </div>
                 </form>
             </div>
```
Comment (Ofarouq310) [inline index.html]: [OPTIONAL]
- [ ] Please move the `form` closing tag one line up, as your div `formtemp` should wrap the whole `form` inside of it.

**Review by Ofarouq310 — APPROVED** [review]
Hi @SnakyMz, ✋ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Descriptive PR title & summary.
✅ No linters errors.
✅ Meaningful commit messages.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (SnakyMz) [reply index.html]: Thank you. I will do it right now.

File: index.html (hunk 3 above)
Comment (SnakyMz) [reply index.html]: My bad will fix it

File: index.html
Comment (SnakyMz) [reply index.html:1]: Thanks a lot. This would have created problems

File: index.html (hunk 4 above)
Comment (SnakyMz) [reply index.html]: Thanks. This explains why I have hard time doing css on it

**Pull request merged**